Frontend/src/Components/Faults/FaultsView.js

```diff
@@ -1,13 +1,334 @@
-import { Heading, HStack, Text, VStack } from "@chakra-ui/react";
+import { Box, Tooltip, Image, SimpleGrid } from "@chakra-ui/react";
+import MPPTContactorImage from "./Images/MPPT Contactor.png";
+import LowContactorImage from "./Images/Low Contactor.png";
+import MotorControllerContactorImage from "./Images/Motor Controller Contactor.png";
+import DoorImage from "./Images/Door.png";
+import BatteryFailsafeImage from "./Images/Battery Failsafe.png";
+import BPSFaultImage from "./Images/BPS Fault.png";
+import EStopImage from "./Images/E-Stop.png";
+import IMDStatusImage from "./Images/IMD Status.png";
+import CrashImage from "./Images/Crash.png";
+import HighCurrentImage from "./Images/High Current.png";
+import HighVoltageImage from "./Images/High Voltage.png";
+import LowCurrentImage from "./Images/Low Current.png";
+import LowVoltageImage from "./Images/Low Voltage.png";
+import LowBatteryImage from "./Images/Low Battery.png";
+import MCUCheckImage from "./Images/MCU Check.png";
+import MPPTCurrentImage from "./Images/MPPT Current.png";
+import PowerWarningImage from "./Images/Power Warning.png";
+import BMSInputVoltageImage from "./Images/BMS Input Voltage.png";
+import PhysicalConnectionImage from "./Images/Physical Connection Lost.png";
+import WirelessCommsLostImage from "./Images/Wireless Comms Lost.png";
+import HighTemperatureImage from "./Images/High Temperature.png";
 
 export default function Faults(props) {
```

Comment:
I found most of the functions in this component pretty confusing, can we write some method headers and/or inline comments?